source/clear-linux/get-started/virtual-machine-install/vmware-esxi-preconfigured-cl-image.rst

```diff
@@ -0,0 +1,245 @@
+.. _vmware-esxi-preconfigured-cl-image:
+
+Run pre-configured Clear Linux image as a VMware\* ESXi guest OS
+################################################################
+
+`VMware ESXi`_ is a type 1 bare-metal hypervisor which runs directly on top of
+server hardware.  With VMware ESXi, you can create, configure, manage, and run a
```

Comment:
delete "a" at the end of this line: "With VMware ESXi, you can create, configure, manage, and run"